include/pa_win_wasapi.h

```diff
@@ -82,7 +82,13 @@ typedef enum PaWasapiFlags
        playback formats that do not match the current configured system settings.
        this is in particular required for streams not matching the system mixer sample rate.
        only applies in Shared mode. */
-    paWinWasapiAutoConvert              = (1 << 6)
+    paWinWasapiAutoConvert              = (1 << 6),
+
+    /* force data output to AC3 passthrough mode */
```

Comment:
Please specify whether the app should pass raw AC3 data or AC3 data wrapped in an S/PDIF wrapper, AKA IEC61937. I think this API requires wrapped IEC61937data so that needs to be clarified.

Please link to an AES or IEC doc or give the exact name of the standard for the data.

Reply:
We're going to merge without resolving this, I created a ticket to track further progress:

https://github.com/PortAudio/portaudio/issues/816

Reply:
If the approach is to defer to Microsoft documentation there should be a comment with a reference to the MS documentation.